packages/x-date-pickers/src/models/adapters.ts

```diff
@@ -153,7 +153,7 @@ export interface MuiPickersAdapter<TDate> {
    * @param {string} format The format to expand.
```

Comment:
I just improved the order of the methods and deprecated `isNull` which is not very usefull.